sig-release/README.md

```diff
@@ -0,0 +1,252 @@
+# SIG Release
+
+### Table of Contents
+- [Mission](#mission)
+- [Join us!](#join-us)
+- [Broad responsibilities](#broad-responsibilities)
+- [Specific responsibilities](#specific-responsibilities)
+- [Relation to the Release Manager and Team](#relation-to-the-release-manager-and-team)
+- [Kubernetes Release Team roles](#kubernetes-release-team-roles)
+   - [Patch Release Manager](#patch-release-manager)
+   - [Kubernetes Release Management Team for Major/Minor Releases](#kubernetes-release-management-team-for-majorminor-releases)
+   - [Release Team Lead](#release-team-lead)
+   - [Release Team Secondary](#release-team-secondary)
+   - [Release Branch Manager](#release-branch-manager)
+   - [Docs Lead](#docs-lead)
+   - [Features Lead](#features-lead)
+   - [Bug Triage Lead](#bug-triage-lead)
+   - [Test Infra Lead](#test-infra-lead)
+   - [Upgrade Testing Lead](#upgrade-testing-lead)
+   - [CI Signal Lead](#ci-signal-lead)
+   - [Release Team Shadow](#release-team-shadow)
+   - [Individual Contributors](#individual-contributors)
+      - [During a patch pelease](#during-a-patch-pelease)
+      - [During a major/minor release](#during-a-majorminor-release)
+         - [Propose and track feature](#propose-and-track-feature)
+         - [Write feature documentation](#write-feature-documentation)
+- [Other activities of the Release Team](#other-activities-of-the-release-team)
+   - [During "Major" releases](#during-major-releases)
+   - [During "Security" releases](#during-security-releases)
+- [References](#references)
+
+## Mission
+- Production of high quality Kubernetes releases on a reliable schedule
+- Ensure there is a [consistent group of community members](https://docs.google.com/document/d/1JAUqKl-lYdYLQ7GUT_9LzqxwQv-PcOdyAxNRZKItajo/edit) [1] in place to support the release process
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+
+## Join us!
```

Comment:
nit: can you add a link to meeting notes.  And possibly create a google doc based off one of the other sigs meeting notes